src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionDescriptorActionApi.java

```diff
@@ -46,20 +48,21 @@ public BlackDuckDistributionDescriptorActionApi(final ContentConverter contentCo
     }
 
     @Override
-    public void validateProviderDistributionConfig(final FieldAccessor fieldAccessor, final Map<String, String> fieldErrors) {
-        final String filterByProject = fieldAccessor.getString(BlackDuckDescriptor.KEY_FILTER_BY_PROJECT).orElse(null);
+    public void validateProviderDistributionConfig(final FieldModel fieldModel, final Map<String, String> fieldErrors) {
+        final String filterByProject = fieldModel.getField(BlackDuckDescriptor.KEY_FILTER_BY_PROJECT).flatMap(field -> field.getValue()).orElse(null);
         if (StringUtils.isNotBlank(filterByProject) && !contentConverter.isBoolean(filterByProject)) {
             fieldErrors.put(BlackDuckDescriptor.KEY_FILTER_BY_PROJECT, "Not a Boolean.");
         }
-        final String projectNamePattern = fieldAccessor.getString(BlackDuckDescriptor.KEY_PROJECT_NAME_PATTERN).orElse(null);
+
+        final String projectNamePattern = fieldModel.getField(BlackDuckDescriptor.KEY_PROJECT_NAME_PATTERN).flatMap(field -> field.getValue()).orElse(null);
         if (StringUtils.isNotBlank(projectNamePattern)) {
             try {
                 Pattern.compile(projectNamePattern);
             } catch (final PatternSyntaxException e) {
                 fieldErrors.put(BlackDuckDescriptor.KEY_PROJECT_NAME_PATTERN, "Project name pattern is not a regular expression. " + e.getMessage());
             }
         }
-        final Collection<String> configuredProjects = fieldAccessor.getAllStrings(BlackDuckDescriptor.KEY_CONFIGURED_PROJECT);
+        final Collection<String> configuredProjects = fieldModel.getField(BlackDuckDescriptor.KEY_CONFIGURED_PROJECT).flatMap(field -> Optional.ofNullable(field.getValues())).orElse(List.of());
```

Comment:
We should make sure FieldValueModel handles nulls better (right now `FieldValueModel.getValue()` could throw an NPE). 
Once that happens, you might be able to convert this: 
`.flatMap(field -> Optional.ofNullable(field.getValues()))` 
to this: 
`.flatMap(FieldValueModel::getValues)`